lib/cc/analyzer.rb

```diff
@@ -15,13 +15,22 @@ module Analyzer
     autoload :Filesystem, "cc/analyzer/filesystem"
     autoload :Formatters, "cc/analyzer/formatters"
     autoload :Issue, "cc/analyzer/issue"
+    autoload :IssueCategoryValidation, "cc/analyzer/issue_category_validation"
+    autoload :IssueCheckNamePresenceValidation, "cc/analyzer/issue_check_name_presence_validation"
+    autoload :IssueDescriptionPresenceValidation, "cc/analyzer/issue_description_presence_validation"
+    autoload :IssuePathExistenceValidation, "cc/analyzer/issue_path_existence_validation"
+    autoload :IssuePathPresenceValidation, "cc/analyzer/issue_path_presence_validation"
+    autoload :IssueRelativePathValidation, "cc/analyzer/issue_relative_path_validation"
     autoload :IssueSorter, "cc/analyzer/issue_sorter"
+    autoload :IssueTypeValidation, "cc/analyzer/issue_type_validation"
+    autoload :IssueValidator, "cc/analyzer/issue_validator"
     autoload :LocationDescription, "cc/analyzer/location_description"
     autoload :LoggingContainerListener, "cc/analyzer/logging_container_listener"
     autoload :MountedPath, "cc/analyzer/mounted_path"
     autoload :RaisingContainerListener, "cc/analyzer/raising_container_listener"
     autoload :SourceBuffer, "cc/analyzer/source_buffer"
     autoload :StatsdContainerListener, "cc/analyzer/statsd_container_listener"
+    autoload :Validation, "cc/analyzer/validation"
```

Comment:
If we move them under that namespace that would work, but these need to be autoloaded under the module to which they belong (in this case, `Analyzer`)